extensions-api/src/main/scala/pl/touk/nussknacker/security/Permission.scala

```diff
@@ -4,11 +4,11 @@ import pl.touk.nussknacker.security
 
 object Permission extends Enumeration {
   type Permission = Value
-  val Read             = Value("Read")
-  val Write            = Value("Write")
-  val Deploy           = Value("Deploy")
-  val Demo             = Value("Demo")
-  val OverrideUsername = Value("OverrideUsername") // Used for migration username forwarding
+  val Read        = Value("Read")
+  val Write       = Value("Write")
+  val Deploy      = Value("Deploy")
+  val Demo        = Value("Demo")
+  val Impersonate = Value("Impersonate") // Used for migration username forwarding
```

Comment:
It's a backward incompatible change. Shouldn't we mention it in the migration guide?

Reply:
Right, I mentioned the change in the migration guide